cmd/tle/commands/commands.go

```diff
@@ -137,6 +137,9 @@ func validateFlags(f *Flags) error {
 		if f.Armor {
 			return fmt.Errorf("-a/--armor can't be used with -d/--decrypt")
 		}
+		if f.Network != defaultNetwork && f.Chain == defaultChain {
```

Comment:
```suggestion
		if f.Network != defaultNetwork && f.Chain != defaultChain {
```

Reply:
Ah! Good catch, thanks

Reply:
actually I think this is correct as-is: this checks if the user has provided a new network via the CLI but _hasn't_ provided new chain hash to go with it. Perhaps another check should be done for the converse though: if they provide another chain hash but the default network, that should error (unless we start a new unchained network in mainnet at some point)

Reply:
`network` is an HTTP endpoint. There might be multiple endpoints serving a given chain.
In this specific case, `defaultNetwork` is `https://api.drand.sh` and the chain is `fastnet`. This means that fastnet can only be served by `https://api.drand.sh`, which I don't think is what we want.

For example, I provide `-n="https://api.drand.sh/<fastnet-hash>"` and `-c=<fastnet-hash>`. This does not pass this check, as network is different from the default network, and the hash is equal to the default network hash.

Reply:
Ahhh true, you raise a good point! I guess we want some additional checks in here

Reply:
I've changed the logic to just emit a warning on stderr when the chain hash isn't changed but the network isn't default.